packages/terra-application-utility/src/utility/_UtilityButton.jsx

```diff
@@ -0,0 +1,131 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import 'terra-base/lib/baseStyles';
+import IconChevronDown from 'terra-icon/lib/icon/IconChevronDown';
+import IconChevronRight from 'terra-icon/lib/icon/IconChevronRight';
+import UtilityMenu from './_UtilityMenu';
+import Utils from '../_Utils';
+import styles from './_UtilityButton.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * The array containing the menu items to be rendered within the menu.
```

Comment:
I am wondering if this correctly displays within the props table?

Reply:
I have similar [questions about this as well](https://github.com/cerner/terra-framework/pull/103/files#r173832493). This particular component doesn't have a prop table for it. Since all of the components share the same props, [only 1 prop table is rendered in the example](https://github.com/cerner/terra-framework/pull/103/files#diff-05e5f45ed82b5665b6f755ee1361d00eR45).